internal/scaffold/ansible/molecule_cluster_create.go

```diff
@@ -17,31 +17,30 @@ package ansible
 import (
 	"path/filepath"
 
-	"github.com/operator-framework/operator-sdk/internal/scaffold"
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
 )
 
-const BuildTestFrameworkAnsibleTestScriptFile = "ansible-test.sh"
+const MoleculeClusterCreateFile = "create.yml"
 
-type BuildTestFrameworkAnsibleTestScript struct {
+type MoleculeClusterCreate struct {
 	StaticInput
 }
 
 // GetInput - gets the input
-func (b *BuildTestFrameworkAnsibleTestScript) GetInput() (input.Input, error) {
-	if b.Path == "" {
-		b.Path = filepath.Join(scaffold.BuildTestDir, BuildTestFrameworkAnsibleTestScriptFile)
+func (m *MoleculeClusterCreate) GetInput() (input.Input, error) {
+	if m.Path == "" {
+		m.Path = filepath.Join(MoleculeClusterDir, MoleculeClusterCreateFile)
 	}
-	b.TemplateBody = buildTestFrameworkAnsibleTestScriptAnsibleTmpl
+	m.TemplateBody = moleculeClusterCreateAnsibleTmpl
 
-	return b.Input, nil
+	return m.Input, nil
 }
 
-const buildTestFrameworkAnsibleTestScriptAnsibleTmpl = `#!/bin/sh
-export WATCH_NAMESPACE=${TEST_NAMESPACE}
-(/usr/local/bin/entrypoint)&
-trap "kill $!" SIGINT SIGTERM EXIT
+const moleculeClusterCreateAnsibleTmpl = `---
 
```

Comment:
Same here with the space; I typically see the document separator with no newline after (but a newline before, if there's a document above):

```yaml
---
some: yaml
```